code/modules/mob/living/living_defense.dm

```diff
@@ -195,6 +195,23 @@
 	adjust_fire_stacks(rand(1,2))
 	IgniteMob()
 
+/mob/living/flamer_fire_act(burnlevel)
+	if(!burnlevel)
+		return
+	if(status_flags & (INCORPOREAL|GODMODE)) //Ignore incorporeal/invul targets
+		return
+	if(hard_armor.getRating(FIRE) >= 100)
+		to_chat(src, span_warning("Your suit protects you from the flames."))
```

Comment:
on living
"suit"

Reply:
"your skin suit protects you from the flames"